libs/dh/wholesale/feature-requests/src/lib/table.ts

```diff
@@ -86,16 +86,16 @@ type Request = ExtractNodeType<GetRequestsDataSource>;
         </ng-container>
 
         <ng-container *wattTableCell="columns['meteringPointTypeOrPriceType']; let row">
-          @if (row.__typename === 'RequestAggregatedMeasureData') {
+          @if (row.__typename === 'RequestCalculatedEnergyTimeSeriesResult') {
             {{ t('meteringPointTypesAndPriceTypes.' + row.meteringPointType) }}
           } @else {
             {{ t('meteringPointTypesAndPriceTypes.' + row.priceType) }}
           }
         </ng-container>
 
         <ng-container *wattTableCell="columns['state']; let row">
-          <watt-badge [type]="row.lifeCycle.statusType">
-            {{ t('states.' + row.lifeCycle.state) }}
+          <watt-badge [status]="row.state">
+            {{ t('states.' + row.state) }}
           </watt-badge>
```

Comment:
suggestion: what we've done in other parts of the app is creating a wrapper component where the `status` property is mapped to badge type. E.g. `dh\libs\dh\market-participant\grid-areas\src\status-badge.component.ts`. This adds an extra layer but since the mapping should be done somewhere, we've done in the app, instead of expanding the `badge`.

Reply:
Good suggestion, I will do that!

Reply:
5f918d55324f1bc747fce85250ce13a07adfb976